hedera-node/src/main/java/com/hedera/services/state/expiry/ExpiringCreations.java

```diff
@@ -148,25 +162,43 @@ public ExpirableTxnRecord saveExpiringRecord(
 			final List<FcAssessedCustomFee> customFeesCharged,
 			final SideEffectsTracker sideEffectsTracker
 	) {
+		return createBaseRecord(memo, receiptBuilder, customFeesCharged, sideEffectsTracker, dynamicProperties.htsDefaultGasCost());
+	}
+
+	private ExpirableTxnRecord.Builder createBaseRecord(
+		final String memo,
+		final TxnReceipt.Builder receiptBuilder,
+		final List<FcAssessedCustomFee> customFeesCharged,
+		final SideEffectsTracker sideEffectsTracker,
+		final Long gasUsed
+	) {
+		// TODO: Populate log info when we have erc20 equivalence.
+		final var contractFunctionResult = ContractFunctionResult.newBuilder()
+			.setContractID(EntityIdUtils.contractParsedFromSolidityAddress(
+				Address.fromHexString(HTS_PRECOMPILED_CONTRACT_ADDRESS)))
```

Comment:
The `contractId` in child records is 359, which is the decimal representation of the hexadecimal address `167`. I think we should convert it to a Hedera account address using the EntityIdUtils methods